### DISCUSSION TITLE
Where is the Teleport config file?

### DISCUSSION THREAD

**Discussion by WilliamLoy** [gravitational/teleport#8153] [Q&A] [answered]
Hi all, 

Can I get a detailed explanation on how to locate the Teleport config file?

Thanks, Will

**Comment by programmerq** [comment, marked as answer]
The default location for the teleport config file is `/etc/teleport.yaml`. If teleport is started without a config file, it will try to start up as a minimal auth/proxy node. You can generate a default config file using `sudo teleport configure -o /etc/teleport.yaml` to get a minimal starting point auth/proxy node. A full config reference can be found here: https://goteleport.com/docs/setup/reference/config/

The teleport config file location can be overridden at runtime by specifying it in the start command. For example, `teleport start -c /path/to/teleport.yaml`. The Docker images that teleport publishes are set to run teleport as `teleport start -c /etc/teleport/teleport.yaml`.
